src/integrationTest/java/uk/gov/hmcts/darts/common/controller/GetWelcomeTest.java

```diff
@@ -24,4 +27,115 @@ void welcomeRootEndpoint() throws Exception {
 
         assertThat(response.getResponse().getContentAsString()).startsWith("Welcome");
     }
+
+    @Test
+    void dailyListAddDailyListEndpoint() throws Exception {
+        MockHttpServletRequestBuilder requestBuilder = post("/dailylist/addDailyList")
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .queryParam("source_system", "CPP")
+            .content("{\n" +
```

Comment:
Would be cleaner if you could move this content to a separate file somewhere?

Reply:
i tried that, and errors appeared in the build. Seems to be some sort of build config problem. thinks any file added is a duplicate.

Reply:
managed to find the setting that was needed to fix the error.